core/templates/dev/head/pages/learner_dashboard/LearnerDashboard.js

```diff
@@ -375,15 +375,26 @@ oppia.controller('LearnerDashboard', [
               if (subsectionName ===
                   LEARNER_DASHBOARD_SUBSECTION_I18N_IDS.EXPLORATIONS) {
               /* eslint-enable max-len */
-                $http.post(
-                  '/learner_dashboard/remove_in_progress_exploration', {
-                    exploration_id: activity.id
+                var removeExpUrl = UrlInterpolationService.interpolateUrl(
+                  '/learnerincompleteactivityhandler/<activityType>' +
+                  '/<activityId>', {
+                    activityType: constants.ACTIVITY_TYPE_EXPLORATION,
+                    activityId: activity.id
                   });
+                /* eslint-disable */
+                $http.delete(removeExpUrl);
```

Comment:
Ah. If you use `$http['delete'](removeExpUrl);`, will that obviate the need for the eslint-disable etc.? Ditto below.